app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -1148,8 +1162,74 @@ async function getFile(fileId, options = {}) {
   }
 }
 
-async function uploadFile(file, config = {}) {
-  return fileService.uploadFile(file, config);
+function uploadFile(file, cfg = {}) {
+  queuedUploadFiles.value.push({
+    file: file,
+    config: {
+      onUploadProgress: cfg.onUploadProgress,
+      headers: cfg.headers,
+    },
+    onUploaded: cfg.onUploaded,
+    onError: cfg.onError,
+  });
+}
+
+async function uploadQueuedFiles() {
+  let err = false;
+  for (let i = queuedUploadFiles.value.length - 1; i >= 0; i--) {
```

Comment:
The reasoning for this was because it was uploading files from the array, and when successful, would remove those files from the array. That way when running the upload file function again, it would just run through the array until it is empty.